db/entities/tableEntity/tableEntity.py

```diff
@@ -0,0 +1,12 @@
+import uuid
+import db.queries.queries as queries
+class TableEntity:
+    id : uuid
+    def __init__(self, table_id: uuid):
+        id=table_id
+        
+    def update():
+        pass
```

Comment:
There is a module named ABC in Python that helps define an abstract class and abstract methods in class with a decorator `@abstractmethod`, seems here update is designed to be one

Reply:
We decided that we are not using the object structure in the end